settings.py

```diff
@@ -1836,7 +1825,6 @@
 
 
 STATIC_UCR_REPORTS = [
-    os.path.join('custom', '_legacy', 'mvp', 'ucr', 'reports', 'deidentified_va_report.json'),
```

Comment:
Not sure if we want to keep the reports that are UCR around as they don't actively keep tech debt around. Its fairly low effort to have a couple of JSON files and a python file for custom expressions (neither of them should depend on other custom code.

Reply:
I'm not sure I understand. Are you suggesting we not delete these UCRs?

Reply:
Would we keep them around thinking that maybe MVP is still using them? I think these projects have been over for long enough that we're better of just removing things and then adding things back on the off chance that someone informs us a particular thing was still being used

Reply:
Ah, okay. I see what you mean, it seems fine to leave in these static UCRs / UCR expressions, I'll add them back